src/transform/sanitize.ts

```diff
@@ -598,9 +598,20 @@ function sanitizeStyles(html: string, options: SanitizeOptions) {
     return styles + content;
 }
 
-export default function sanitize(html: string, options?: SanitizeOptions) {
+export default function sanitize(
+    html: string,
+    options?: SanitizeOptions,
+    forcedSanitizeCssWhiteList?: CssWhiteList,
```

Comment:
1. I suggest replacing it with an interface that has an additional parameter for uniformity and future expansion of functionality.
```
    html: string,
    options?: SanitizeOptions,
    additionalOptions?: SanitizeOptions,
```
2. Then it will be:
```
    if (forcedSanitizeCssWhiteList) {
        sanitizeOptions.cssWhiteList = {
            ...sanitizeOptions.cssWhiteList,
            ...additionalOptions.cssWhiteList,
        };
    }
```